ReSharper.FSharp/src/FSharp.Psi.Features/src/Util/FSharpAttributesUtil.fs

```diff
@@ -48,3 +48,14 @@ let addAttributesList (decl: IFSharpTreeNode) addNewLine =
         else
             Whitespace()
     ] |> ignore
+
+let getAttributeList typeDecl =
+    let typeDeclarationGroup = TypeDeclarationGroupNavigator.GetByTypeDeclaration(typeDecl)
```

Comment:
Since it only works for type declarations, it'd be better to show it in the name (so it wouldn't be mistakenly used on other declarations).